src/mon/OSDMonitor.h

```diff
@@ -446,6 +449,12 @@ class OSDMonitor : public PaxosService {
       pending_inc.new_flags &= ~flag;
     }
   }
+  epoch_t get_epoch_recent_pool_delete() {
+    return epoch_recent_pool_delete;
+  }
+  void set_epoch_recent_pool_delete(epoch_t e) {
+    epoch_recent_pool_delete = e;
+  }
```

Comment:
This is problematic.. it'll only be set on teh primary, which means it won't be correct if there is a election and leader change or a mon restarts